katas/content/superposition/parity_bitstrings/solution.md

```diff
@@ -0,0 +1,56 @@
+We are going to use a recursive approach that was hinted at in the task.
+
+Let's denote the required state on $N$ qubits as $|P_{N,0}\rangle$ for `parity = 0` and $|P_{N,1}\rangle$ for `parity = 1`. We can group the $2^{N-1}$ basis states included in the state $|P_{N,0}\rangle$ by their first bit ($0$ or $1$) and write the state as follows:
+
+$$|P_{N,0}\rangle = \frac{1}{\sqrt{2^{N-1}}} \sum_{k : k \text{ has parity }0} |k\rangle_N = \\\\
```

Comment:
\\\\ don't work in plain formulas right now (only in special environments like matrix or cases), so let's make each line a separate formula instead